flopy/plot/plotutil.py

```diff
@@ -2754,12 +2770,15 @@ def intersect_modpath_with_crosssection(
     nppts = {}
 
     for cell, verts in projpts.items():
+        tcell = cell
+        while tcell >= ncpl:
+            tcell -= ncpl
```

Comment:
@jlarsen-usgs  Isn't this equal to `tcell = cell % ncpl` (modulo, remainder after division)?

Reply:
Yes, this statement is equivalent to `tcell = cell % ncpl`